packages/schemas/tables/application_organization_scope_relations.sql

```diff
@@ -0,0 +1,14 @@
+/* init_order = 2 */
+
+/** The organization scopes (permissions) assigned to an application. */
+create table application_organization_scope_relations (
```

Comment:
This may be confusing with application_roles as they are representing different meanings but have similar names?

Reply:
Updated